app/routes/api/items/$id.js

```diff
@@ -0,0 +1,62 @@
+import { Item } from "~/../models/Item";
+
+export const action = async ({request, params}) => {
+  try {
+    switch (request.method) {
+      case "GET":
+        try {
+          const item = Item.query().findById(id);
+          return {
+            statusCode: 200,
+            body: JSON.stringify(item),
+          };
+        } catch(err) {
+          return {
+            statusCode: 500,
+            body: "GET is a failure!",
+          };
+        }
```

Comment:
Can the method ever be `GET` here? I thought `GET` requests are handled in the `loader`. Maybe if there is no `loader`, the `action` gets invoked as a fallback?